Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -242,10 +243,88 @@ double HGCalDDDConstants::cellSizeHex(int type) const {
   return cell;
 }
 
+double HGCalDDDConstants::distFromEdgeHex(double x, double y, double z) const {
```

Comment:
@bsunanda - please, add a comment what is calculated here. Is the point(x,y) inside of the polygon?

Reply:
@bsunanda it takes longer for me to check the code without comment on what it suppose to do.